tests/sentry/issues/test_producer.py

```diff
@@ -124,7 +124,7 @@ def test_payload_sent_to_kafka_with_partition_key(
         mock_produce.assert_called_once_with(
             ArroyoTopic(name="ingest-occurrences"),
             KafkaPayload(
-                bytes(occurrence.fingerprint[0], "utf-8"),
+                occurrence.fingerprint[0].encode(),
                 json.dumps({"mock_data": "great"}).encode("utf-8"),
```

Comment:
For consistency, this (and other occurrences) should be changed from `.encode("utf-8")` -> `.encode()` too, IMO.

Reply:
agreed, but I'm only fixing the (nonsense) calls to `bytes(...)` in this PR